modules/claims_api/app/controllers/claims_api/v2/blueprints/auto_established_claim_blueprint.rb

```diff
@@ -9,8 +9,12 @@ class AutoEstablishedClaimBlueprint < Blueprinter::Base
           'forms/526'
         end
 
-        field :attributes do |claim, _options|
-          claim&.form_data
+        field :attributes do |claim, options|
+          if options[:async] == false
+            { claimId: claim&.evss_id&.to_s }.merge(claim&.form_data)
+          else
+            claim&.form_data
+          end
```

Comment:
testing in local and the `claimId` is null again. looks like you removed the needed reload to refresh the db entry? adding a reload resolves it.